cip25-wasm/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cardano-multiplatform-lib-cip25",
+  "version": "3.1.1",
+  "description": "Multiplatform SDK for CIP25 (Cardano NFT Metadata)",
+  "scripts": {
+    "rust:build-nodejs": "rimraf ./pkg; wasm-pack build --target=nodejs; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:build-browser": "rimraf ./pkg; wasm-pack build --target=browser; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:build-web": "rimraf ./pkg; wasm-pack build --target=web; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:publish": "cargo publish",
```

Comment:
Do we really want to publish the WASM repo? Or are we instead trying to publish the non-WASM crate?

Reply:
I thought we wanted to publish the WASM crates to NPM and the rust ones to crates.io? I just copied what we had in the old build script but with a per-crate (or at least per end-user crate) approach. I don't know how the actual process works after that though since it was always done by someone else which is why I asked if there was a problem with having multiple packages referring to one repo.